src/types.ts

```diff
@@ -117,6 +117,12 @@ export type TurboTokenPriceForBytesResponse = {
   token: TokenType;
 };
 
+export type TurboFiatEstimateForBytesResponse = {
```

Comment:
Let's expand this type to include the amount of winc that would be expended in the operation as well.